E2E/pageObjects/game.page.ts

```diff
@@ -57,11 +60,24 @@ export = {
   },
 
   checkCopyLinkButton: () => {
-    I.waitForElement(locator.buttons.copyLink);
-    I.click(locator.buttons.copyLink);
+    I.waitForElement(locator.buttons.copyLinkHeader);
+    I.click(locator.buttons.copyLinkHeader);
     I.waitForElement(locator.text.linkCopiedToClipboard);
     I.waitForInvisible(locator.text.linkCopiedToClipboard);
-    I.click(locator.buttons.copyLink);
+    I.click(locator.buttons.copyLinkHeader);
     I.waitForElement(locator.text.linkCopiedToClipboard);
   },
+
+   getPlayersList: async () => {
```

Comment:
this method has misleading name.

It should either return actual list of player objects or be renamed to indicate, that it returns some sort of a count.